src/js/view/controls/settings-menu.js

```diff
@@ -12,11 +12,10 @@ import {
 
 export function createSettingsMenu(controlbar, onVisibility, localization) {
     const settingsButton = controlbar.elements.settingsButton;
-    const onMenuEmpty = () => {
-        settingsButton.hide();
-    };
-
-    const settingsMenu = SettingsMenu(onVisibility, onMenuEmpty, localization);
+    const settingsMenu = SettingsMenu(onVisibility, {
+        hide: () => settingsButton.hide(),
+        show: () => settingsButton.show()
```

Comment:
Is this hiding all settings or just the rate tab in the settings?

Ideally we only hide/show or add/remove the tab, and the menu button is shown or hidden based on the number of settings tabs.

Reply:
I get it now.